pkgs/by-name/pa/pangolin_0_6/package.nix

```
@@ -0,0 +1,64 @@
+{ stdenv, lib, fetchFromGitHub, cmake, pkg-config, doxygen, glew
+, xorg, ffmpeg_4, libjpeg, libpng, libtiff, eigen
+, Carbon ? null, Cocoa ? null
```

```suggestion
, Carbon, Cocoa
```
This doesn't work and the `? null` just hide that. You must use darwin.apple_sdk.frameworks.Cocoa instead.